packaging/PLATFORM_SUPPORT.md

```diff
@@ -0,0 +1,153 @@
+  effort to ensure that things keep working on them either. Additionally, Netdata may occasionally move an
+  older supported platform to this tier if we can no longer provide full support for it for technical reasons.
+  Users reporting bugs on previously supported platforms will be asked to reproduce the issue on a currently
+  supported platform, and if they cannot the issue will be closed.
+
+Any platforms not listed in any of these categories may or may not work
+
+> ### What happens when a platform becomes ‘Previously Supported’ for technical reasons?
+>
+> In general, if we determine that we need to drop support for a given platform due to technical limitations,
+> this will be announced in the release notes of the next stable release with a deprecation notice. The platform
+> will be supported for _that release_, and will be removed from nightlies some time before the next release.
+
+> ### What constitutes ‘End of Life’ for a platform?
+>
+> We consider a platform to be end-of-life when the upstream maintainers of that platfrom stop providing official
+> support for it themselves, or when that platform transitions into an ‘extended security maintenance’ period.
+>
+> Platforms that meet these criteria will be immediately transitioned to the **Previously Supported** category,
+> with no prior warning from Netdata and no deprecation notice, unlike those being dropped for technical reasons,
+> as our end of support should already coincide with the end of the normal support lifecycle for that platform..
+
+A general outline of the various support tiers and categories is shown in the following table
+
+|   | Bug Support| Guaranteed Configurations | CI Coverage | Native Packages | Static Build Support |
+| - | -------------------- | ------------------------- | ----------- | -------------- | -------------------- |
+| Core | High priority | Everything but rare edge cases | Full | Yes, if we can provide them | Full |
+| Intermediate | Normal priority | Common cases | Partial (CI mostly equivalent to **Core**, but possibly with some gaps, and not required to pass) | Possibly | Full |
+| Community | Best Effort | Default only | None | No | Best Effort |
+| Third-party Supported | Users directed to platform maintainers | None | None | No | Best Effort |
+| Previously Supported | Users asked to upgrade | None | None | Yes, but only already published versions | Best Effort |
+
+- ‘Bug Support’: How we handle of platform-specific bugs.
+- ‘Guaranteed Configurations’: Which runtime configurations for the agent we try to guarantee will work with minimal effort from users.
+- ‘CI Coverage’: What level of coverage we provide for the platform in CI.
+- ‘Native Packages’: Whether we provide native packages for the system package manager for the platform.
+- ‘Static Build Support’: How well our static builds are expected to work on the platform.
+
+# Currently supported platforms
+
+## Core
```

Comment:
I would prefer a matrix of all supported platforms with a column that states what kind of supported package is supported (native, static, build from source) and whether this is provided by netdata or not.

Reply:
That’s what we had previously. It was confusing to users, a pain in the arse to read even if you understood what it meant, and was a serious pain in the arse to update. Getting rid of that complexity was the whole point of codifying a completely new support policy.

And, the support tier is supposed to (barring rare, special exceptions) clearly delineate this information. The only unusual bit here is that we only provide static builds for Linux systems (but that doesn’t matter given that more than 99% of our user base is using Linux).